packages/flutter/test/material/drawer_test.dart

```diff
@@ -106,4 +106,36 @@ void main() {
 
     semantics.dispose();
   });
+
+  testWidgets('Drawer scrimDrawerColor test', (WidgetTester tester) async {
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Scaffold(
+          drawerScrimColor: const Color(0xFF323232),
+          drawer: Drawer(),
+        ),
+      ),
+    );
+
+    final ScaffoldState state = tester.firstState(find.byType(Scaffold));
+    state.openDrawer();
+
+    await tester.pump();
+    await tester.pump(const Duration(seconds: 1));
+
+    final Container container = _getContainerScaffold(tester);
+    final BoxDecoration decoration = container.decoration;
+    expect(decoration.color, const Color(0xFF323232));
+    expect(decoration.shape,  BoxShape.rectangle);
+  });
 }
+
+Container _getContainerScaffold(WidgetTester tester) {
```

Comment:
There's no need to factor this out. It could be a local function within the one test that needs it, or it could be just be inlined there.

Reply:
okay :), done